src/components/layout/Footer/Footer.tsx

```
@@ -26,8 +32,39 @@ export const Footer = (): React.ReactElement => {
                 </a>
               </li>
               <li>
-                <a href="#download" className={styles.link}>
-                  Download
+                <a href="#faq" className={styles.link}>
+                  Pricing
+                </a>
+              </li>
+              <li>
+                <a href="#" className={styles.link}>
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)